### PR TITLE
versions

### PR DESCRIPTION
**Pull request by faddat**
See #1099 for discussion of the issues here.  This PR addresses some of these, as well as this

https://github.com/crypto-org-chain/chain-main/pull/682

﻿- Updates to core cosmos library versions
  - tendermint 0.34.15
  - sdk 0.44.5
  	- fixes root.go
  - tm-db 0.6.6
  - ibc-go v2.0.2

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,35 +3,36 @@ module github.com/cosmos/gaia/v6
...
 	github.com/gorilla/mux v1.8.0
 	github.com/gravity-devs/liquidity v1.4.2
 	github.com/rakyll/statik v0.1.7
 	github.com/spf13/cast v1.4.1
 	github.com/spf13/cobra v1.2.1
 	github.com/strangelove-ventures/packet-forward-middleware v1.0.1
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.14
-	github.com/tendermint/tm-db v0.6.4
+	github.com/tendermint/tendermint v0.34.15
```
Comment (yaruwangway) [inline go.mod:15]: tendermint 0.34.15 need key migration. We need to integrate [pre-upgrade](https://github.com/cosmos/cosmos-sdk/issues/9973) to the app ?

Comment (williambanfield) [reply go.mod:15]: Hey @yaruwangway! I think that the change you're referring to is the data base key change that occurred as part of [v0.35](https://github.com/tendermint/tendermint/blob/a97b081df100cb43398fabb925ffae3f35b9816a/UPGRADING.md#database-key-format-changes). Which means that that change does not play a role in updating to Tendermint v0.34.15.

The change log for v0.34.15 was mostly [bug fixes](https://github.com/tendermint/tendermint/blob/41c176ccc6a75d25631d0f891efb2e19a33329dc/CHANGELOG.md#L11)

Comment (yaruwangway) [reply go.mod:15]: ha, right! thanks!

File: go.mod [hunk 2]
```diff
@@ -3,35 +3,36 @@ module github.com/cosmos/gaia/v6
 go 1.17
 
 require (
-	github.com/cosmos/cosmos-sdk v0.44.3
-	github.com/cosmos/ibc-go/v2 v2.0.0
+	github.com/cosmos/cosmos-sdk v0.44.5
```
Comment (yaruwangway) [inline go.mod]: I think this break the `cmd/gaiad/cmdroot.go`

 `client.ReadHomeFlag`  -> `client.ReadPersistentCommandFlags`

Comment (faddat) [reply go.mod]: It does.  I've fixed it several times, I'll fix it again now :).

Comment (yaruwangway) [reply go.mod]: Hi @faddat , I can checkout a branch off your branch to fix it, I need to bump versions to test the `pre-upgrade`  anyway. 😁

Comment (faddat) [reply go.mod]: sure, I've got it fixed now.

**Comment by tac0turtle** [comment]
@faddat could you resolve the conflicts, then we can merge

**Comment by faddat** [comment]
sure! sorry I missed that.

**Review by tac0turtle — APPROVED** [review]

**Review by okwme — APPROVED** [review]

**Pull request merged**